library/lists.inc

```diff
@@ -73,9 +73,16 @@ function collect_issue_type_category() {
 }
 
 // Build the $ISSUE_TYPES array (see script header for description)
-$res = sqlStatement("SELECT * FROM `issue_types` WHERE active=1 AND `category`=? ORDER BY `ordering` ASC", array( collect_issue_type_category() ));
+$res = sqlStatement("SELECT * FROM `issue_types` WHERE active = 1 AND `category`=? ORDER BY `ordering`",
+  array(collect_issue_type_category()));
 while($row = sqlFetchArray($res)){
-  $ISSUE_TYPES[$row['type']] = array(xl($row['plural']),xl($row['singular']),xl($row['abbreviation']),$row['style'],$row['force_show']);
+  $ISSUE_TYPES[$row['type']] = array(
```

Comment:
I think this may be a good example of having a potential scope issue when you bring this script above inside your function. If you bring it into the function first then the $ISSUE_TYPES will be limited to within the function. So, if then try to include_once it outside the function after this, will end up with an empty $ISSUE_TYPES.